src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -73,7 +73,8 @@ public class FileResource {
     private final UserRepository userRepository;
 
     // NOTE: this list has to be the same as in file-uploader.service.ts
-    private final List<String> allowedFileExtensions = new ArrayList<>(Arrays.asList("png", "jpg", "jpeg", "svg", "pdf", "zip"));
+    private final List<String> allowedFileExtensions = new ArrayList<>(Arrays.asList("png", "jpg", "jpeg", "svg", "pdf", "zip", "odt", "ods", "odp", "odg", "odf", "doc", "docx",
+            "csv", "xls", "xlsx", "ppt", "pptx", "pages", "rtf", "pages-tef", "numbers", "key"));
```

Comment:
```suggestion
    private final List<String> allowedFileExtensions = List.of("png", "jpg", "jpeg", "svg", "pdf", "zip", "odt", "ods", "odp", "odg", "odf", "doc", "docx",
            "csv", "xls", "xlsx", "ppt", "pptx", "pages", "rtf", "pages-tef", "numbers", "key");
```

(Probably requires some more reformatting for the line break within the arguments list to satisfy the automatic formatter.)

Reply:
Thank you, its done

Reply:
It does not work as it is an immutable collection then and causes an error on other place